pkg/framework/proxies.go

```diff
@@ -0,0 +1,249 @@
+package framework
+
+import (
+	"context"
+	"errors"
+
+	configv1 "github.com/openshift/api/config/v1"
+	mapiv1beta1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog"
```

Comment:
I think we should be using `klog/v2` now
```suggestion
	"k8s.io/klog/v2"
```

Reply:
this still needs to be adressed

Reply:
thanks.  i'll take care of that today.